### DISCUSSION TITLE
Unable to get audio from teensy/audio board output jack

### DISCUSSION THREAD

**Discussion by dgtl311** [ElectroTechnique/electrotechnique.github.io#19] [Hardware Issues]
I very carefully followed the build instructions and perfectly soldered the audio board on the teensy and still cannot get any audio out of the audio jack. I tried using two different sets of headphones and even amplified speakers and don't get any sound at all. I tried playing audio and MIDI files from three different computers (two Windows 10 and one Mac). The TSynth shows up as an audio device on all computers and even graphically shows that it is playing audio but I get no audio at all out of the jack on the audio board. All of the volume levels are up and the device is enabled so everything software wise checks out. I triple checked all of the soldering and even re-flashed the teensy with the firmware but still nothing. Does anyone have any ideas on why I would not be getting any audio? I don't want to continue the build until I can confirm that the teensy/audio board are working. Photos showing the two boards soldered  are attached. Thank you!
![659920F2-5BD0-4376-822D-24E4CE8934D3](https://user-images.githubusercontent.com/79732668/109372658-53a62600-7870-11eb-9a0b-935ef3cafb86.jpg)
![D6F55DC7-FB3D-4E6C-9132-9F23C73629EA](https://user-images.githubusercontent.com/79732668/109372660-57d24380-7870-11eb-9e50-fc3b1ba8749b.jpg)

**UPDATE**: I confirmed that the audio teensy/audio board are working by uploading the following Teensy-Synth_Part_1 code:
https://drive.google.com/file/d/1PpIdpE4J-3jRJIueRORdcDn9rGRK01VK/view

I was able to hear the intended audio from the output jack into an amplified speaker so I will continue the build and hope for the best!

**Comment by woll** [comment]
The build guide is a bit vague on exactly how to test the audio out of the teensy/audio board before installing. 
On a Mac I attached the Teensy via USB to the Mac and used vmpk (a midi output piano app) and the Audio Recording feature of Quicktime Player (select the TSynth as the input) to be able to hear the audio output. The output was silent until Midi notes were played (and then it was very random and could be quite quiet).

**Reply by dgtl311** [reply]
Thank you for the advice! I'm almost done with the build - just pending LEDs, attaching teensy, jumper wires, knobs and case. But I will give what you suggested a try and let you know how it goes. Thanks again!
